src/main.ts

```diff
@@ -24,13 +24,17 @@ export default async function run() {
         const inputs = getInputs();
         const azureSqlAction = new AzureSqlAction(inputs);
         
-        const runnerIPAddress = await SqlUtils.detectIPAddress(inputs.connectionConfig);
-        if (runnerIPAddress) {
-            let azureResourceAuthorizer = await AuthorizerFactory.getAuthorizer();
-            let azureSqlResourceManager = await AzureSqlResourceManager.getResourceManager(inputs.connectionConfig.Server, azureResourceAuthorizer);
-            firewallManager = new FirewallManager(azureSqlResourceManager);
-            await firewallManager.addFirewallRule(runnerIPAddress);
+        // Unless skip-firewall-check is set to true, check if the runner's IP address is allowed to connect to the server
+        if (core.getBooleanInput('skip-firewall-check') !== true) {
```

Comment:
noting that the input parameter isn't parsed into the IActionConfig like the rest of the parameters but this is ok since we're not pulling it into further functions or debugging output

Reply:
That's a good point, I'll add it to IActionInputs just to be consistent